src/executors/ExecutorTeamCity.js

```diff
@@ -12,42 +12,72 @@ export class ExecutorTeamCity extends Executor {
         this.runFunctions = {};
```

Comment:
runFunctions and the 'run' function can be removed, as we have gone back to just calling 'execute' for now.

Reply:
Thanks @wyvern8 

Will get the suggested changes in and update the request.